src/utils/jira/getAssignedJiraTickets.ts

```diff
@@ -0,0 +1,17 @@
+import axios from "axios";
+import { backendURL } from "../../constants";
+
+export default async function getAssignedJiraTickets({
+  userEmail,
+}: {
+  userEmail: string;
+}) {
+  const assignedJiraTickets = await axios.post(
+    `${backendURL}/api/jira/getAssignedTicketsInProgress`,
+    {
+      userEmail,
+    }
+  ).then(res => res.data);
```

Comment:
What happens on error?

Reply:
What do you suggest should happen?

Reply:
Tell the user? Failing silently might be a problem. 
Also, should we get a report?

Reply:
Ok

Reply:
I added a block of code that sends report to the telemetry system. Not sure if this is the way to do it since other utils files don't have such. Does this look reasonable?